jobs/build/ose/scripts/merge-and-build.sh

```diff
@@ -4,6 +4,41 @@
 
 set -o xtrace
 
+# Checks the master and stage branch of the repo for the Version in the specified spec filename.
+# Calculates a higher version string which can be used with tito.
+function getPostStageVersion {
```

Comment:
functions name should be lower-cased and with underscores to separate words.. so `get_post_stage_versions()`